pkg/bloomcompactor/bloomcompactor.go

```diff
@@ -583,29 +457,50 @@ func (c *Compactor) runCompact(ctx context.Context, logger log.Logger, job Job,
 	}
 
 	if len(metas) == 0 {
-		// Get chunks data from list of chunkRefs
-		chks, err := storeClient.chunk.GetChunks(ctx, makeChunkRefs(job.Chunks(), job.Tenant(), job.Fingerprint()))
+		localDst := createLocalDirName(c.cfg.WorkingDirectory, job)
+		defer func() {
+			//clean up the bloom directory
+			if err := os.RemoveAll(localDst); err != nil {
+				level.Error(logger).Log(fmt.Sprintf("failed to remove block directory %s", localDst), err)
```

Comment:
We structure our error logs so they contain a `msg` field and `err` field:

```suggestion
				level.Error(logger).Log("msg", "failed to remove block directory", "dir", localDst, "err", err)
```

Reply:
Please check occurrences further down the line